SieveofEratosthenes.py

```diff
@@ -0,0 +1,9 @@
+// The one-liner uses the reduce function to remove, one step at a time,
+//all “marked” numbers from the initial set of all numbers between 2 and n (in the one-liner: set(range(2, n))).
```

Comment:
Kindly comment properly. In python, Use # for single line comment and triple quotes (" " ") for multiline comments

Reply:
done